regions/io/core.py

```diff
@@ -29,12 +65,118 @@ def to_regions(self):
                 msg = 'Skipping elliptical annulus {}'.format(shape)
                 warn(msg, DS9RegionParserWarning)
                 continue
+            if shape.region_type in ['box', 'symbol', 'text'] and shape.format_type == 'CRTF':
+                msg = 'Skipping {} {}'.format(shape.region_type, shape)
+                warn(msg, CRTFRegionParserWarning)
+                continue
             log.debug(shape)
             region = shape.to_region()
             log.debug(region)
             regions.append(region)
         return regions
 
+    def to_crtf(self):
+        pass
```

Comment:
should this `return self` instead?  I'm not sure what the intent of this method is.

Reply:
this is the method to convert to CRTF strings, have not implemented in this PR.

Reply:
Then instead of `pass`, `raise NotImplementedError("This method will do XXYY")`